packages/coinstac-api-server/src/data/resolvers.js

```diff
@@ -42,6 +42,30 @@ initSubscriptions(pubsub);
 const resolvers = {
   JSON: GraphQLJSON,
   Query: {
+    /**
+     * Returns all results.
+     * @return {array} All results
+     */
+    fetchAllResults: () => fetchAll('run'),
```

Comment:
This is good. Will take some finessing when I merge this with the integration branch as I'm filtering runs by member/owner.